cirq-google/cirq_google/serialization/serializable_gate_set.py

```diff
@@ -322,10 +328,10 @@ def deserialize(
                 constants=proto.constants,
                 deserialized_constants=deserialized_constants,
             )
-            return circuit if device is None else circuit.with_device(device)
+            if device is not None:
+                circuit._device = device  # coverage: ignore
+            return circuit
         if which == 'schedule':
-            if device is None:
-                raise ValueError('Deserializing schedule requires a device but None was given.')
```

Comment:
There is a break here that schedules can now have no device. Since we aren't using these anymore I figured this was an acceptable vendor break. @dstrain115

Reply:
SG